games/guess-it/README.md

```diff
@@ -0,0 +1,34 @@
+## Scrypto Unit: A lightweight testing framework for Scrypto
```

Comment:
this file is somewhat unnecessary, but it was helpful in helping me get scrypto-unit up and running properly

Reply:
Could you change the contents of the `README.md` so that it includes some instructions on how people can play the game? Like what sort of functions and methods need to be called in what order and perhaps an example.